components/script/dom/node.rs

```diff
@@ -1420,6 +1421,15 @@ pub(crate) unsafe fn from_untrusted_node_address(candidate: UntrustedNodeAddress
     DomRoot::from_ref(Node::from_untrusted_node_address(candidate))
 }
 
+/// If the given untrusted node address represents a valid DOM node in the given runtime,
+/// returns it.
+#[allow(unsafe_code)]
+pub(crate) unsafe fn from_untrusted_compositor_node_address(
+    candidate: CompositorUntrustedNodeAddress,
+) -> DomRoot<Node> {
+    DomRoot::from_ref(Node::from_untrusted_compositor_node_address(candidate))
+}
+
```

Comment:
We had to make a new version of functions in this file because the `webrender_traits` (really should be called `compositor_traits`) crate has its own version of `UntrustedNodeAddress` and now we are passing hit test results straight through to the `ScriptThread` from the compositor.